models/user.go

```diff
@@ -83,18 +83,24 @@ type User struct {
 	Email            string `xorm:"NOT NULL"`
 	KeepEmailPrivate bool
 	Passwd           string `xorm:"NOT NULL"`
-	LoginType        LoginType
-	LoginSource      int64 `xorm:"NOT NULL DEFAULT 0"`
-	LoginName        string
-	Type             UserType
-	OwnedOrgs        []*User       `xorm:"-"`
-	Orgs             []*User       `xorm:"-"`
-	Repos            []*Repository `xorm:"-"`
-	Location         string
-	Website          string
-	Rands            string `xorm:"VARCHAR(10)"`
-	Salt             string `xorm:"VARCHAR(10)"`
-	Language         string `xorm:"VARCHAR(5)"`
+
+	// MustChangePassword is an attribute that determines if a user
+	// is to change his/her password after registration.
+	// PS : This scenario only plays out if an admin created the account
+	MustChangePassword bool `xorm:"INDEX"`
```

Comment:
Why `INDEX` this column?
Should be same as in migration `xorm:"NOT NULL DEFAULT false"`

Reply:
Not familiar with xorm and it broke earlier ( https://drone.gitea.io/go-gitea/gitea/2097 )... Would fix